src/plopp/graphics/interactive.py

```diff
@@ -81,23 +81,23 @@ def home(self):
         self.crop(**self._crop)
         self.draw()
 
-    def pan(self):
+    def pan(self, *_):
```

Comment:
Silently ignoring other args? Why and where is this required?

Reply:
I wanted the `ToggleButton` tool to send its current value when it is pressed, so that the `PointsTool` can decide whether to start or stop.

In this particular case, we use the tool state from the matplotlib figure to determine which zoom or pan tool should be active. So we don't actually care about the `value` of the tool for `pan` and `zoom` (we could care for `logx`, `logy` but not necessarily, cf the current implementation).

That said, in the current implementation the `PointsTool` is actually a `ToggleTool` and it knows its current value, so I could just do
```Py
    def start_stop(self):
        if self.value:
            self.points.start()
        else:
            self.points.stop()
```
and no need to send the value when the button is toggled.

I guess I was thinking of future toggle tools, that might call a method that is not part of the class, and that might need the value in the callback message.

But that could be in the future. I am happy to just remove the value from the call, and remove the `*_` from the `fig.py`.

Reply:
That, or at least add some documentation about what is going on.